### PR TITLE
add hypershift-mce-aws-destroy-2.2

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

File: ci-operator/step-registry/hypershift/mce/aws/destroy/2.2/hypershift-mce-aws-destroy-2.2-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        
+        bin/hypershift dump cluster --artifact-dir=$ARTIFACT_DIR \
+          --dump-guest-cluster=true \
+          --namespace local-cluster \
+          --name="${CLUSTER_NAME}"
+      
+        echo "$(date) Deleting HyperShift cluster ${CLUSTER_NAME}"
+        bin/hypershift destroy cluster aws \
+          --aws-creds=${CLUSTER_PROFILE_DIR}/.awscred  \
+          --name ${CLUSTER_NAME} \
```
Comment (dtthuynh) [inline ci-operator/step-registry/hypershift/mce/aws/destroy/2.2/hypershift-mce-aws-destroy-2.2-chain.yaml:26]: do we not need the infra-id argument here as well?

Comment (heliubj18) [reply ci-operator/step-registry/hypershift/mce/aws/destroy/2.2/hypershift-mce-aws-destroy-2.2-chain.yaml:26]: no need to set infra-id when destroying a hosted cluster  [Delete a HostedCluster](https://hypershift-docs.netlify.app/getting-started/#:~:text=%3D%24NODEPOOL_REPLICAS-,Delete%20a%20HostedCluster,-To%20delete%20a)

**Review by rokej — APPROVED** [review]

**Pull request merged**